src/ui/inspector/StylesInspectorView.vue

```diff
@@ -24,16 +25,38 @@ export default {
         this.openmct.selection.off('change', this.updateSelection);
     },
     methods: {
+        getStyleProperties(item) {
+            let styleProps = {};
+            for (let key in item) {
```

Comment:
Another place to consider `Object.keys()`. Up to you though.

Reply:
Yep. Done.